dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionValidation.java

```diff
@@ -0,0 +1,156 @@
+/*
+ *    Copyright 2018 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package io.dockstore.webservice.core;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.ComparisonChain;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import javafx.util.Pair;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+/**
+ * This describes the validation information associated with one or more files for a version
+ * @author aduncan
+ * @since 1.6.0
+ */
+@ApiModel("VersionValidation")
+@Entity
+@Table(name = "versionvalidation")
+@SuppressWarnings("checkstyle:magicnumber")
+public class VersionValidation implements Comparable<VersionValidation> {
```

Comment:
If you go with my idea of including import information (not sure if you should), then maybe add it here? Although it's not really validation data... Not sure.

Reply:
I do think that maybe some of this code is eventually lifted out and done asynchronously. There is potential for a more lengthy "processing workflow" to be queued and done outside of the webservice.

Reply:
Ya it could be a column here, I think that makes sense to store the information together. Perhaps it should be called something else instead of validation, something that includes import information.